src/ray/object_manager/plasma/store_runner.cc

```diff
@@ -133,7 +133,11 @@ void PlasmaStoreRunner::Start() {
   ArrowLog::ShutDownArrowLog();
 }
 
-void PlasmaStoreRunner::Stop() { loop_->Stop(); }
+void PlasmaStoreRunner::Stop() {
+  if (loop_) {
```

Comment:
When can this happen (where the `if` is needed)?

Reply:
I don't know the causes yet, but it happens on my machine on Windows.

Reply:
Then it's a bug. Can you document that there is a bug here and that this is a workaround but we don't understand why it is necessary (along with instructions for how to reproduce it)?